lang/inputs.go

```diff
@@ -318,6 +318,12 @@ func inputCode(s string, fs engine.Fs) (*ParsedInput, error) {
 		}
 	}
 
+	// Check if code is `metadata.yaml`, which is obviously not correct here
```

Comment:
nit: Please either add a period at the end of the comment so it's a regular sentence, or uncapitalize the first letter.

Full explanation:
https://github.com/purpleidea/mgmt/blob/master/docs/style-guide.md#comments